Components/IModel.cs

```diff
@@ -98,6 +98,12 @@ public interface IModel
         /// </summary>
         List<FooterSection> FooterSections { get; set; }
 
+        /// <summary>
+        /// Used to display a custom header menu on the top of the page for app templates
+        /// Set by application programmatically
```

Comment:
We should probably add a mention similar to FooterSections above that it is only available in the application template for GGCweb theme.  (I forgot to mention that for the banner :) but these comments can end up as documentation for users (at least in Java they do))

Reply:
Should be a separate issue looool jk jk I'll make the change :)

Reply:
done!

Reply:
:)